product_unique_serial/tests/test_for_unicity.py

```diff
@@ -27,7 +27,7 @@
 from copy import deepcopy
 
 from openerp import exceptions
-from . import test_common import TestCommon
+from openerp.addons.product_unique_serial.tests.test_common import TestCommon
```

Comment:
I mean, Why use `openerp.addons.product_unique_serial.tests.test_common` instead of `. test_common`?